my-first-react-app/src/StoryBox.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+class StoryBox extends React.Component {
+	render() {
+		return(<div>My Story Box Component</div>);
+	}
```

Comment:
In this component you have different tab spacing than in the rest of your app. Check your settings in ATOM, it should be 2 spaces. But eslint should also tell you in the other project.